src/org/zaproxy/zap/extension/httpsinfo/ExtensionHttpsInfo.java

```diff
@@ -83,16 +87,17 @@ public void hook(ExtensionHook extensionHook) {
 		if (getView() != null) {
 			// Register our popup menu item, as long as we're not running as a
 			// daemon
-			MenuEntry entry = getPopupMsgMenuExample();
+			MenuEntry entry = getPopupMsgMenu();
```

Comment:
This "helper" variable could be removed, the method call could be added as parameter of the other method.